### ISSUE TITLE
Please provide a migration guide for deprecated and removed methods

### ISSUE THREAD

**Issue by benjamindamm** [bpmn-io/bpmn-js-properties-panel#674]
Hi bpmn-js-Team

I am trying hard to migrate existing code to your new version of bpmn-js. The huge problem is, that I am using dmn-js, camunda as well. So the situation is a bit more complex than a default setup. I am very much confused tbh. Are you going to replace dmn-js-properties-panel as well under the hood of this project?

For me its very hard to understand how to replace all the functions I am already using. Let me give you some examples:

1. registerProvider
2. entryFactory.selectBox (I am using this to inject translations)

How can I migrate stuff like this? As I can see many of the old functions had been removed. Is there a possibility to provide some advice how to replace those functions?

Kind regards
Benjamin Damm

**Comment by nikku** [comment]
Hi @benjamindamm, thanks for approaching us on this matter. You're feedback is valueable and others may come across such questions, too.

We'll look into the two specific scenarios you mentioned and report back.

**Comment by nikku** [comment]
CC @barmac

**Comment by benjamindamm** [comment]
Thanks a lot for your fast response. We could approach a little migration guide. If you need any specific details which methods I am using feel free to contact.

Btw I am using Angular. It could be possible that some things are related to this fact. But as I can see right know the problems are more of a general issue.

**Comment by barmac** [comment]
Hi @benjamindamm,

Have you read [the breaking changes section](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/CHANGELOG.md#breaking-changes) of the 1.0.0 release? We've provided links to two migration pull requests which should be a good starting point. We are also planning to have a blog post on the release, so if there's anything which should be explained there in your opinion, feel free to share it.

> Are you going to replace dmn-js-properties-panel as well under the hood of this project?

We have also released version 1.0.0 of dmn-js-properties-panel which is based on [the new properties panel framework](https://github.com/bpmn-io/properties-panel), if that's what you're asking about.
